actors/builtin/miner/miner_test.go

```diff
@@ -2509,26 +2514,33 @@ func makeProveCommit(sectorNo abi.SectorNumber) *miner.ProveCommitSectorParams {
 }
 
 func makeFaultParamsFromFaultingSectors(t testing.TB, st *miner.State, store adt.Store, faultSectorInfos []*miner.SectorOnChainInfo) *miner.DeclareFaultsParams {
-	//deadlines, err := st.LoadDeadlines(store)
-	//require.NoError(t, err)
-	faultAtDeadline := make(map[uint64][]uint64)
-	// TODO minerstate
-	// Find the deadline for each faulty sector which must be provided with the fault declaration
-	//for _, sectorInfo := range faultSectorInfos {
-	//	dl, p, err := miner.FindSector(deadlines, sectorInfo.SectorNumber)
-	//	require.NoError(t, err)
-	//	faultAtDeadline[dl] = append(faultAtDeadline[dl], uint64(sectorInfo.SectorNumber))
-	//}
-	params := &miner.DeclareFaultsParams{Faults: []miner.FaultDeclaration{}}
-	// Group together faults at the same deadline into a bitfield
-	for dl, sectorNumbers := range faultAtDeadline {
-		fault := miner.FaultDeclaration{
-			Deadline: dl,
-			Sectors:  bitfield.NewFromSet(sectorNumbers),
+	deadlines, err := st.LoadDeadlines(store)
```

Comment:
I thought at one point I might need to get faults working in tests. I did, but ended up not needing the faults in the end. This restoration of test code should be a leg up for getting fault tests back up.